python/ccxt/async/base/exchange.py

```diff
@@ -165,6 +165,6 @@ def add_new_tokens(self):
 
     async def update_order(self, id, symbol, *args):
```

Comment:
We don't have an update_order %)

Reply:
Your intent was to `edit_order` here, probably )

Reply:
The current async base class has update order. I just changed how errors were raised but good spot I will also change that now :)